pkg/util/node/node.go

```diff
@@ -33,7 +33,7 @@ import (
 func GetHostname(hostnameOverride string) string {
 	hostname := hostnameOverride
 	if string(hostname) == "" {
```

Comment:
I think the `string` casts can be removed
hostname may be directly assigned with the `=` operator instead of `:=`, which leads to a `var err error` before, but I think it's worth it